src/test/java/com/trycloud/stepDefinitions/LogIn_stepDef_TD.java

```diff
@@ -13,9 +13,11 @@
 public class LogIn_stepDef_TD {
     Logger LOG = LogManager.getLogger();
     LoginPage loginPage=new LoginPage();
-    Actions action=new Actions(Driver.getDriver());
+
     @Given("the user is on the login page")
     public void the_user_is_on_the_login_page() {
+        String URL =Driver.getDriver().getCurrentUrl();
```

Comment:
variables should be named in lowercase, unless they are constant variables, which should have final keyword